velox/functions/sparksql/aggregates/AverageAggregate.cpp

```diff
@@ -129,16 +166,25 @@ exec::AggregateRegistrationResult registerAverage(
                   AverageAggregate<int32_t, double, double>>(resultType);
             case TypeKind::BIGINT: {
               if (inputType->isShortDecimal()) {
-                return std::make_unique<DecimalAverageAggregateBase<int64_t>>(
-                    resultType);
+                auto [p, s] = getDecimalPrecisionScale(*inputType.get());
```

Comment:
The s is not used, auto precision = getDecimalPrecisionScale(*inputType.get()).first;